docs/getting-started/summaryofchanges.md

```diff
@@ -56,6 +60,21 @@ The following features and enhancements were added.
 - Added a more helpful and actionable description to message ZWEAM511E, which occurs when API ML does not trust the certificate provided by the service. [#818](https://github.com/zowe/api-layer/issues/818)
 
 #### Zowe App Server
+- Added option to write JSON files without using Node.js. [#137](https://github.com/zowe/zlux-app-server/pull/137)
```

Comment:
It's specific to a certain action.

"The install-app.sh script used to install App Server plugins can now be used without Node.js. If Node.js is not detected when the script is executed, this behavior will be automatic. But, you can also force this behavior with the environment variable `INSTALL_NO_NODE=1`, such as in `INSTALL_NO_NODE=1 ./install-app.sh ~/zlux-editor`